src/services/comment.services.js

```diff
@@ -19,7 +19,7 @@ async function postComment(octokit, owner, repo, issueOrPullRequestNumber, comme
       body: comment,
     });
 
-    console.log(`Comment posted successfully on Issue/PR #${issueOrPullRequestNumber}`);
+    console.log(`Comment posted successfully.`);
```

Comment:
Because on a previous log, I am referencing the number of the PR. Below is an example:

![image](https://github.com/BigSamu/OpenSearch_Changelog_Workflow/assets/28614451/2b1e8e13-0b98-4909-959f-63ed724d523f)